libnetdata/dyn_conf/README.md

```diff
@@ -119,16 +119,18 @@ Where:
 #### REPORT_JOB_STATUS
 
 ```
-REPORT_JOB_STATUS {MODULE_NAME} {JOB_NAME} {STATUS} {STATE} "{REASON}"
+REPORT_JOB_STATUS {MODULE_NAME} {JOB_NAME} {STATUS} {STATE} ["REASON"]
```

Comment:
```suggestion
REPORT_JOB_STATUS {MODULE_NAME} {JOB_NAME} {STATUS} {STATE} [{REASON}]
```

Why `"`?

Reply:
most probably reason will be more than single word and therefore must be in quotes